src/pages/vedtak/vedtaksoppsummering-nb.ts

```diff
@@ -0,0 +1,39 @@
+export default {
+    'behandlet.av': 'Saksbehandler',
+    'attestert.av': 'Attestant',
+
+    'vurdering.tittel': 'Vurdering',
+    'vurdering.innvilgelse': 'Innvilgelse',
+    'vurdering.avslag': 'Avslag',
+
+    'behandling.ikkeFerdig': 'Behandlingen er ikke ferdig',
+    'behandling.søknadsdato': 'Søknadsdato',
+    'behandling.saksbehandlingsdato': 'Saksbehandlingsdato',
+
+    'brev.utkastVedtaksbrev': 'Utkast til vedtaksbrev',
+
+    'feilmelding.behandlingIkkeFerdig': 'Behandlingen er ikke ferdig',
+    'feilmelding.errorkode': 'Error kode:',
+    'feilmelding.fantIkkeBehandlingsId': 'Fant ikke behandlingsid',
```

Comment:
Irrelevant for bruker at det er ID vi ikke klarte å finne. Kall det bare "behandling" i stedet.